crowbar_framework/spec/models/api/crowbar_spec.rb

```diff
@@ -125,6 +125,36 @@
     end
   end
 
+  context "with enough compute resources" do
+    it "succeeds to find enough compute nodes" do
+      allow(NodeObject).to(
+        receive(:find).with("roles:nova-compute-kvm").
+        and_return([NodeObject.find_node_by_name("node1"), NodeObject.find_node_by_name("node2")])
```

Comment:
thanks for adding the tests, but you have to use `[NodeObject.find_node_by_name("testing.crowbar.com"), NodeObject.find_node_by_name("testing.crowbar.com")]`, as we only have one node and the admin node in the fixtures.